finagle/h2/src/main/java/io/netty/handler/codec/http2/BetterHttp2ConnectionEncoder.java

```diff
@@ -28,6 +28,7 @@
 import static io.netty.handler.codec.http2.Http2CodecUtil.DEFAULT_PRIORITY_WEIGHT;
 import static io.netty.handler.codec.http2.Http2Error.PROTOCOL_ERROR;
 import static io.netty.handler.codec.http2.Http2Exception.connectionError;
+import static io.netty.handler.codec.http2.Http2FrameWriter.Configuration;
```

Comment:
I have a slight preference for omitting this import and changing the reference to this class on line 324 to be:

```
    public Http2FrameWriter.Configuration configuration() {
```

That more consistent with how it is referenced on line 83:

```
        Http2FrameWriter.Configuration config = configuration();
```